docs/developer-docs/integrations/sns/integrate-sns/ledger-integration.md

```diff
@@ -1 +1,69 @@
-Title h1: SNS Ledger Integration
\ No newline at end of file
+# Ledger canister
+
+The ledger canister keeps track of the transactions between accounts.
```

Comment:
It also (explicitly) keeps track of all the accounts' balances, right? Should we add this? 
Similarly to above, I would maybe add one sentence why we talk about the ledger in the SNS documentation:
- ledger can be deployed and used by itself (I think this is already explained on another page, maybe we link to there?)
- as part of the SNS canisters, a ledger canister is always deployed
- the ledger canister follows the ICRC-1 standard (maybe a link to the standard / code?)

Reply:
Indeed it keeps track of the account balances at the last block of the chain.